src/dos/program_imgmount.cpp

```diff
@@ -212,13 +212,20 @@ void IMGMOUNT::Run(void) {
         return;
     }
 
+    std::vector<std::string> files;
     // find all file parameters, assuming that all option parameters have been removed
     while (cmd->FindCommand((unsigned int)(paths.size() + 2), temp_line) && temp_line.size()) {
         // Try to find the path on native filesystem first
         const std::string real_path = to_native_path(temp_line);
         if (real_path.empty()) {
```

Comment:
get_expanded_files is doing this same work on the `temp_line` string in it's first couple lines:

``` c++
	const std::string real_path = to_native_path(path);
	if (!real_path.empty()) {
		files.push_back(real_path);
		return true;
	}
```

Reply:
The said lines in get_expanded_files were originally intended for more general case (so that the function may also be called elsewhere when needed), but for this case these lines in get_expanded_files will now be skipped to avoid repeated calls.